### PR TITLE
Auth UI tweaks

### PR DESCRIPTION
**Pull request by seve**
Addresses changes asked for in #1900 

Screen Shot 2020-10-07 at 2 06 04 PM
Screen Shot 2020-10-07 at 2 06 35 PM
Screen Shot 2020-10-07 at 2 06 42 PM

### CODE REVIEW

File: client/src/components/leftSidebar/infoMenu.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          <MenuItem
+            target="_blank"
+            text={
+              libraryVersions && libraryVersions.cellxgene
+                ? libraryVersions.cellxgene
+                : null
+            }
+          />
+          <MenuItem text="MIT License" />
+          {tosURL ? (
```
Comment (tihuan) [inline client/src/components/leftSidebar/infoMenu.js:42]: I think we don't need ternary if the alternative is `null`!

E.g., `{ tosURL && <MenuItem href={tosURL} target="_blank" text="Terms of Service" /> }`

File: client/src/components/leftSidebar/infoMenu.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            href="https://github.com/chanzuckerberg/cellxgene"
+            target="_blank"
+            icon="git-branch"
+            text="Github"
+            rel="noopener"
+          />
+          <MenuItem
+            target="_blank"
+            text={
+              libraryVersions && libraryVersions.cellxgene
```
Comment (tihuan) [inline client/src/components/leftSidebar/infoMenu.js:36]: suggest `text={ libraryVersions?.cellxgene || null }`

File: client/src/components/leftSidebar/infoMenu.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            <MenuItem href={tosURL} target="_blank" text="Terms of Service" />
+          ) : null}
+          {privacyURL ? (
+            <MenuItem
+              href={privacyURL}
+              target="_blank"
+              text="Privacy Policy"
+              rel="noopener"
+            />
+          ) : null}
```
Comment (tihuan) [inline client/src/components/leftSidebar/infoMenu.js:52]:
```
{ privacyURL && (<MenuItem
              href={privacyURL}
              target="_blank"
              text="Privacy Policy"
              rel="noopener"
            />)}
```

File: client/src/components/menubar/authButtons.js [hunk 4]
```diff
@@ -1,30 +1,80 @@
...
+  Tooltip,
+  Popover,
+  Menu,
+} from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+
 import * as globals from "../../globals";
 import styles from "./menubar.css";
 
+const baseEmoji = [0x1f9d1, 0x1f468, 0x1f469];
```
Comment (tihuan) [inline client/src/components/menubar/authButtons.js:15]: really cool!! Super inclusive 😆 💯 👍

Comment (tihuan) [inline client/src/components/menubar/authButtons.js:15]: These are constants, let's use ALL_CAPS!

File: client/src/components/menubar/authButtons.js [hunk 5]
```diff
@@ -1,30 +1,80 @@
...
 
+  const scientist = String.fromCodePoint(
+    baseEmoji[sexIndex],
+    skinTones[skinToneIndex],
+    zeroWidthJoiner,
+    microscope
+  );
+
+  if (!auth?.["requires_client_login"]) return null;
+  if (userinfo?.["is_authenticated"])
```
Comment (tihuan) [inline client/src/components/menubar/authButtons.js:35]: For readability, let's wrap the code block with `{}`

like so:

```ts
if (userinfo?.["is_authenticated"]) {
  return ( ... )
}
```

**Review by tihuan — APPROVED** [review]
Looks really good! Just some tiny nits. Otherwise LGTM 🚀 🏅 !! Thanks so much, @seve !

**Comment by tihuan** [comment]
Also quick feedback, I added screenshots this time, but for UI related PRs in the future, can we please include screenshots with the PR 😄 🙏 ? That'll help reviewers tremendously in knowing where to look when they spin up the server locally, instead of digging through the code to figure out where changes are made! Also adding QA steps will help too 👍 

Thank you!

File: client/src/components/leftSidebar/infoMenu.js (hunk 2 above)
Comment (lesliecodes) [reply client/src/components/leftSidebar/infoMenu.js:36]: ➕ to above

File: client/src/components/leftSidebar/infoMenu.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+          />
+          <MenuItem
+            href="https://github.com/chanzuckerberg/cellxgene"
+            target="_blank"
+            icon="git-branch"
+            text="Github"
+            rel="noopener"
+          />
+          <MenuItem
+            target="_blank"
```
Comment (lesliecodes) [inline client/src/components/leftSidebar/infoMenu.js]: can we also add
```
rel="noopener"
```

File: client/src/components/leftSidebar/infoMenu.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+            text={
+              libraryVersions && libraryVersions.cellxgene
+                ? libraryVersions.cellxgene
+                : null
+            }
+          />
+          <MenuItem text="MIT License" />
+          {tosURL ? (
+            <MenuItem href={tosURL} target="_blank" text="Terms of Service" />
```
Comment (lesliecodes) [inline client/src/components/leftSidebar/infoMenu.js:36]: add rel="noopener" to any target="_blank" links

File: client/src/components/leftSidebar/infoMenu.js (hunk 6 above)
Comment (seve) [reply client/src/components/leftSidebar/infoMenu.js]: Weird, I just cut this out of the original info menu. I could of sworn you made these changes when merging Colin's PR a few weeks ago.

Comment (lesliecodes) [reply client/src/components/leftSidebar/infoMenu.js]: merge gone bad maybe lol

File: client/src/components/leftSidebar/infoMenu.js (hunk 7 above)
Comment (tihuan) [reply client/src/components/leftSidebar/infoMenu.js:36]: ooh eagle eye! Thanks Leslie! Done!

**Comment by tihuan** [comment]
Just updated the PR! PTAL again @lesliecodes @colinmegill 🙏 Thank you!

**Review by lesliecodes — APPROVED** [review]
LGTM!

File: client/src/components/leftSidebar/infoMenu.js (hunk 6 above)
Comment (seve) [reply client/src/components/leftSidebar/infoMenu.js]: ¯\_(ツ)_/¯ I'll add them in rn :)

**Pull request merged**